services/web/serve-static.js

```diff
@@ -1,9 +1,14 @@
-const fs = require('fs');
-const url = require('url');
 const Koa = require('koa');
-const compress = require('koa-compress');
-const koaStatic = require('koa-static');
-const config = require('@kaareal/config');
+const koaMount = require('koa-mount');
```

Comment:
for later. I am wondering if we shouldnt just drop serve-static and put nginx in front of this? 
Like the serve-static was always meant as short term hack :)

Reply:
Interesting, I'm down with that!

Reply:
Putting this on the back burner for now but I like the idea